client/sites/components/panel-sidebar/style.scss

```diff
@@ -13,15 +14,26 @@
 .panel-sidebar {
 	width: 160px;
 	flex-shrink: 0;
-
-	ul {
-		list-style: none;
-	}
 }
 
 .panel-sidebar-tab {
+	border-radius: 4px;
 	width: 100%;
+	margin: 2px 0;
+	padding: 8px;
+	height: 40px;
 	color: var(--studio-gray-100);
+	white-space: nowrap;
+	font-weight: 400;
+	font-size: $font-body-small;
```

Comment:
The sidebar text looks a bit too large now; based on figma it should be 13px, could we check? (It was already 13px before this change)

<img width="551" alt="image" src="https://github.com/user-attachments/assets/c535704b-fe5a-4718-8b71-7a17bf969bfc">

Reply:
Figma looks like a bug, it should be using typography variables https://github.com/Automattic/wp-calypso/blob/trunk/packages/typography/styles/variables.scss#L46

Reply:
I'm fine either way, just pointing out this inconsistency, we can iterate later, no worries 😄